doc/developer.md

```diff
@@ -32,6 +32,30 @@ You can run tests by invoking `make test` in the sourmash directory;
 `python -m pytest` will run the Python tests, and `cargo test` will
 run the Rust tests.
 
+### If you're having trouble installing the development environment
+
+Some installation issues can be solved by simply removing the intermediate build files with: 
+
+```
+make clean
+```
+
+If you are getting an error that contains `ImportError: cannot import name 'to_bytes' from 'sourmash._minhash'`, then it's likely you need to update Rust and clean up your environment. 
+
+Additionally, the Rust language is a fast-moving target and the latest version may be required, which can be acquired with 
+
+```
+rustup update
+```
```

Comment:
We test for MSRV (minimum supported rust version) on CI, and at the moment it's [1.34][0] (which is fairly old).
So I think this can be rephrased to have the `make clean` instructions, but not necessarily the `rustup update` parts.

[0]: https://github.com/dib-lab/sourmash/blob/0362ac3e1ba10600bb6a5b90410b7930b925ae37/.github/workflows/rust.yml#L219L243